aggregates/src/commonMain/kotlin/org/dhis2/mobile/aggregates/model/CellInfo.kt

```diff
@@ -1,6 +1,7 @@
 package org.dhis2.mobile.aggregates.model
 
 import org.dhis2.mobile.aggregates.ui.states.InputExtra
+import org.hisp.dhis.mobile.ui.designsystem.component.LegendData
```

Comment:
so here there is a break dependency rule, entities (model) shouldn't know about ui frameworks